wpilibc/src/main/native/cpp/DataLogManager.cpp

```diff
@@ -129,6 +129,8 @@ void Thread::Main() {
         }
         auto size = entry.file_size();
         if (fs::remove(entry.path(), ec)) {
+          fmt::print(stderr, "DataLogManager: Deleted {}",
```

Comment:
stderr does not show up in the ds console as errors or warnings, if that was the intention here.

Reply:
If I use `FRC_ReportError`, what error code should I use?